test/test_external.py

```diff
@@ -246,11 +244,9 @@ def assertIsFile(path):
 
 class TestLoadFromPipeline(unittest.TestCase):
     def test_question_answering(self):
```

Comment:
Suggest renaming this method to something like `test_text_to_text_model_from_pipeline` now